.rubocop.yml

```diff
@@ -48,6 +48,8 @@ Layout/ExtraSpacing:
 Layout/MultilineMethodCallIndentation:
   Enabled: true
   EnforcedStyle: indented
+Layout/EndOfLine:
+  Enabled: False
```

Comment:
I forgot about this
Let's keep the `rubocop.yml `file intact let's not add anything